documentation/CONFIG.md

```diff
@@ -158,6 +175,39 @@ Connector-specific configuration properties are described below.
   * Type: String
   * Importance: Low
   * Default: kafka_connect_offsets
+  
+###Topic to Table
+
+These configurations can be specified for multiple Kafka topics from which records are being processed. 
+Also, these topic level configurations will be override the behavior of Connector level configurations such as 
+``scylladb.consistency.level``, ``scylladb.deletes.enabled`` and ``scylladb.ttl``
+
+``topic.my_topic.my_ks.my_table.mapping``
+
+  For mapping topic and fields from Kafka record's key, value and headers to ScyllaDB table and its columns.
+  
+  **Note**: Ensure that the data type of the Kafka record's fields are compatible with the data type of the ScyllaDB column.
+  In the Kafka topic mapping, you can optionally specify which column should be used as the ttl (time-to-live) and 
+  timestamp of the record being inserted into the database table using the special property __ttl and __timestamp.
+  By default, the database internally tracks the write time(timestamp) of records inserted into Kafka. 
```

Comment:
why is all of this text bold??

Reply:
Not sure why? This is all fine in intellJ. We haven't used anything that will make this text bold.

Reply:
The "__" string is why: https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax

Reply:
For mapping kafka record values to ttl and timestamp, they have to pass it as `__ttl` or "__timestamp".